### ISSUE TITLE
Distribute as Flatpak on Flathub

### ISSUE THREAD

**Issue by felipecrs** [benjamimgois/goverlay#202]
As there was no other issue open already, I'm opening this one. Some advantages are:

- Simplified installation: no need to add repositories, run scripts or anything. Just go to the Store and install.
- Compatibility with Steam Deck without the need of disabling the read-only root file system.
- Always up to date on any distro. Currently, the version available on PPA is 0.6.1 (while the latest is 0.8.1).

## References

https://github.com/benjamimgois/goverlay/issues/165#issuecomment-996910851

**Comment by ChaosBlades** [comment]
Yeah, something needs to be done about Ubuntu based distros, flatpak or otherwise. They are the most common linux installs and the current release is about 8 months old. At this point I doubt it will get updated again.

**Comment by benjamimgois** [comment]
Hey guys, i know... flatpak is a high priority project now. I'm just out of time to play on the project lately, but it will happen !
